src/platform/plugins/shared/share/public/url_service/redirect/redirect_manager.ts

```diff
@@ -35,8 +35,11 @@ export class RedirectManager {
     application.register({
       id: 'r',
       title: 'Redirect endpoint',
-      chromeless: true,
```

Comment:
@tsullivan @eokoneyo the main idea of this PR is to show chrome when navigating in-between apps through the `/r/` app. I can't think of any bad side-effects why we shouldn't show it even on initial render, as even on the initial load via `/r/` endpoint we most likely will redirect to an app with chrome enabled. 

A safer, but a more complex fix could be if we'd track inside core if this is an initial render or an in-between apps redirect. We'd show chrome if we're navigating in-between-apps, and hide it if this is initial rendering. But this more would become more complex. 


WDYT? any reason we shouldn't do it?

Reply:
@Dosant The main idea sounds perfectly reasonable. I think it would help to add some videos to this PR so we can examine the results, though. It would help me understand what you mean by "initial render" (does this mean right after login?)

Reply:
By initial render I meant when you just open a kibana, or a page reload.
I'll do some recordings

Reply:
@tsullivan see description for videos. suggest to download and scrub through

Reply:
Thanks for the videos! They do show that the changes give a much smoother look and faster feel